doc/manual.rst

```diff
@@ -545,7 +545,7 @@ is optional if it is not ambiguous (only hexadecimal floating-point literals
 with a type suffix can be ambiguous).
 
 
-The type suffixes are:
+The pre-defined type suffixes are:
```

Comment:
open question: could we add `proc f64(a: string): float64` etc to system.nim to make those non-builtin?
what would this break? beside fixing issues like https://github.com/timotheecour/Nim/issues/125